README.md

```diff
@@ -1,2 +1,50 @@
+![Small Logo](static/img/polaris-logo.png)
+
 # polaris-schema
-Library that helps you create polaris based schema
+
+[![Build Status](https://travis-ci.com/Enigmatis/polaris-schema.svg?branch=master)](https://travis-ci.com/Enigmatis/polaris-schema)
+[![NPM version](https://img.shields.io/npm/v/@enigmatis/polaris-schema.svg?style=flat-square)](https://www.npmjs.com/package/@enigmatis/polaris-schema)
+
+#### Install
+
+```
+npm install polaris-schema
+```
+
+### Overview
+
+This library helps you to set the fundamental definitions of the polaris based schema that you are going to create.
```

Comment:
This library helps you set all of the fundamental definitions of polaris based schema.